docs/reference/errors-and-warnings/NU1107.md

```diff
@@ -29,5 +29,7 @@ To install a specific version, see the information for the tool that you're usin
 - [nuget.exe CLI](../../consume-packages/install-use-packages-nuget-cli.md#install-a-specific-version-of-a-package)
 - [Package Manager Console](../ps-reference/ps-ref-install-package.md)
 
+Give our new experimental one-click tool, [NuGetSolver](https://devblogs.microsoft.com/nuget/introducing-nugetsolver-a-powerful-tool-for-resolving-nuget-dependency-conflicts-in-visual-studio/), a try. This tool can automatically fix this problem if you’re using VS2022 with nuget.org. Please provide us with feedback. Based on the feedback received, we will decide whether to officially support this new tool and incorporate it into the NuGet package manager.

```

Comment:
This sounds a bit like a telemarketer's pitch :D 
Not sure if that's necessarily bad, but it caught me off-guard personally. 

Here's another idea: 

> 
Use NuGetSolver, our experimental tool for solving dependency conflicts, to help you find solutions to problems like this automatically!
If you're using Visual Studio 2022, and have nuget.org as your only source, [NuGetSolver](https://devblogs.microsoft.com/nuget/introducing-nugetsolver-a-powerful-tool-for-resolving-nuget-dependency-conflicts-in-visual-studio/) can help free your dependency graph of resolver warnings.
[Learn more about how you can try out NuGetSolver](https://devblogs.microsoft.com/nuget/introducing-nugetsolver-a-powerful-tool-for-resolving-nuget-dependency-conflicts-in-visual-studio/#usage-recommendations-and-known-constraints) and we'd love to [hear your feedback](https://devblogs.microsoft.com/nuget/introducing-nugetsolver-a-powerful-tool-for-resolving-nuget-dependency-conflicts-in-visual-studio/#feedback) about your experience.

My logic was: 

First sentence to provide info on what it is. 2nd to communicate constraints, and then instructions. 

We don't need to worry too much about the whole incorporating it within nuget package manager automatically, since we're already calling it experimental.

Reply:
> This sounds a bit like a telemarketer's pitch :D

Apologies if it came across that way. In the past, I received feedback on other PRs to be concise and direct. I wanted to clarify here that this is an experiment, without creating any expectations that `NuGet.Solver` is an officially supported tool. `NuGet.Solver` was published under the `Microsoft DevLab` account to clearly indicate that it is not supported unless there is a strong demand for it.